src/Symfony/Component/ErrorHandler/Resources/bin/patch-type-declarations

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env php
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+if (\in_array('-h', $argv) || \in_array('--help', $argv)) {
+    echo implode(PHP_EOL, [
+        ' Patches type declarations to fix incompatibility with Symfony 6.',
```

Comment:
Note that the script isn't bound to Symfony 6. It works for all deps in vendor/ (that might be important for the doc also: this is part or a generic deprecation framework for return types)